i18n/en/docusaurus-plugin-content-docs/current/guides/examples/entities.mdx

```diff
@@ -0,0 +1,47 @@
+---
+sidebar_position: 5
+---
+
+# Entities
+
+`entities` layer in Feature-Sliced Design encapsulates core business logic, similar to entities in backend applications. However, its use in frontend applications is not always necessary and should be approached with caution to avoid overcomplicating the codebase.
+
+## When to Use the `entities` Layer
+
+Most frontend applications function as "thin clients" with minimal business logic, relying on the backend for data processing. In such cases, `entities` layer can often be omitted in favor of `shared/api`. Use `entities` layer only when the following conditions are met:
```

Comment:
praise: great that you mention thin clients, this concept is very important to proper use of Entities, but is currently absent in the docs

suggestion: I do feel like it might be a non-self-explanatory concept though, so how about we add a proper `##` heading explaining what are thin vs thick clients before this section?

Reply:
Done